app/js/data/iconListInfo.js

```diff
@@ -0,0 +1,62 @@
+// JSON Object of all of the icons and their tags
+
+const icons = {
```

Comment:
More idiomatic here would just be to export the object itself, and skip giving it a name.